classes/ObjectModel.php

```diff
@@ -675,7 +675,17 @@ public function update($null_values = false)
                 // A little explanation of what we do here : we want to create multishop entry when update is called, but
                 // only if we are in a shop context (if we are in all context, we just want to update entries that alread exists)
                 $shop_exists = Db::getInstance()->getValue('SELECT '.$this->def['primary'].' FROM '._DB_PREFIX_.$this->def['table'].'_shop WHERE '.$where);
+                if (Shop::isFeatureActive()) {
+                    if (Shop::getContext() != Shop::CONTEXT_SHOP) {
+                        $display_multishop_checkboxes = true;
```

Comment:
Hello @fatmaBouchekoua 
WDYT about : 

``` php 
$display_multishop_checkboxes = Shop::isFeatureActive() && Shop::getContext() != Shop::CONTEXT_SHOP;
```

Like that, is always defined for your next condition
ping @maximebiloe